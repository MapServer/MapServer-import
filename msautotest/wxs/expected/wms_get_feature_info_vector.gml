Content-type: application/vnd.ogc.gml

<?xml version="1.0" encoding="ISO-8859-1"?>

<msGMLOutput 
	 xmlns:gml="http://www.opengis.net/gml"
	 xmlns:xlink="http://www.w3.org/1999/xlink"
	 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
	<rivers_layer>
		<rivers_feature>
			<gml:boundedBy>
				<gml:Box srsName="EPSG:4326">
					<gml:coordinates>-88.910118,37.042652 -77.650139,42.224796</gml:coordinates>
				</gml:Box>
			</gml:boundedBy>
			<NAME>Ohio</NAME>
			<SYSTEM>Mississippi</SYSTEM>
		</rivers_feature>
	</rivers_layer>
</msGMLOutput>
