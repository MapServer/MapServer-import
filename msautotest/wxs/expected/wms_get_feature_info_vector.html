Content-type: text/html

<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
        <head>
                <meta http-equiv="Content-Type" content="text/html;charset=iso-8859-1" />
                <meta http-equiv="Content-Style-Type" content="text/css" />
                <meta http-equiv="pragma" content="no-cache"/>
                <meta http-equiv="Expires" content="Thu, 01 Dec 1994 120000 GMT"/>
				<link rel="stylesheet" media="all" href="/ms_ogc_workshop/demo.css" type="text/css"/>
                <title>Query Results</title>
        </head>
        <body>
                <table class="centered" border="0">
                        <tr>
                                <td>Query Details</td>
                        </tr>
                </table>

                <hr/>

                <table class="centered" border="1">
                        <tr>
                                <th>Bounding box</th>
                                <td>-179.640000,-179.640000,179.640000,179.640000</td>
                        </tr>
                        <tr>
                                <th>Image Coordinates</th>
                                <td>141, 91</td>
                        </tr>
                        <tr>
                                <th>Map Coordinates</th>
                                <td>-78.323040, 35.282000</td>
                        </tr>
                        <tr>
                                <th>Number of results</th>
                                <td>1</td>
                        </tr>
                        <tr>
                                <th>Scale</th>
                                <td>565834182.163200</td>
                        </tr>
                        <tr>
                                <th>Cellsize</th>
                                <td>1.796400</td>
                        </tr>
                </table>

                <hr/>
		<table class="centered" border="0">
			<tr>
				<td>Query Results for Layer rivers</td>
			</tr>
		</table>
		<hr/>
		<table class="centered" border="1">
			<tr>
				<th>Result</th>
				<th>Name</th>
				<th>System</th>
			</tr>
			<tr>
				<td>1</td>
				<td>Ohio</td>
				<td>Mississippi</td>
			</tr>
                </table>
				<hr/>
                <table class="centered" cellpadding="3" cellspacing="0" border="0">
                        <tr>
                                <td><img src="/cgi-bin/mapserv.exe?map=tomk_workshop.map&amp;mode=querymap&amp;img.x=141&amp;img.y=91&amp;imgext=-180 -90 180 90&amp;layer=land_shallow_topo_2048&amp;" width="400" height="300" alt="Querymap" title="Querymap"/></td>
                        </tr>
                        <tr>
                                <th colspan="2" align="center">standard querymap</th>
                        </tr>
                </table>
                <hr/>
                <p>
                        <a href="http://validator.w3.org/check?uri=referer" title="[ Valid XHTML 1.0! ]"><img class="flat" src="http://www.w3.org/Icons/valid-xhtml10" alt="[ Valid XHTML 1.0! ]" height="31" width="88"/></a>
                        <a href="http://jigsaw.w3.org/css-validator/check/referer" title="[ Valid CSS! ]"><img class="flat" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="[ Valid CSS! ]" height="31" width="88"/></a>
                </p>
        </body>
</html>